.github/workflows/check-generated.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.22.2"
+          go-version: "1.22.3"
```

Comment:
[optional] we can save ourselves from doing this in the future if we instead use https://github.com/celestiaorg/celestia-app/blob/ce1453f6e706ad25d09e2346070bdf79718f867d/.github/workflows/test.yml#L11-L13

Reply:
sure, let's do it in a separate PR and backport it to v0.34.x-celestia

Reply:
SGTM